src/app/pages/registration/register-user.jsx

```diff
@@ -59,15 +72,19 @@ function ChplRegisterUser({ hash }) {
           },
           onError: (error) => {
             if (error.status === 401) {
-              setMessage('A user may not have more than one role, or your username / password are incorrect');
+              enqueueSnackbar('A user may not have more than one role, or your username / password are incorrect', {
+                variant: 'error',
+              });
             } else {
-              setMessage(error.response.data.error);
+              enqueueSnackbar(error.response.data.error, {
+                variant: 'error',
+              });
             }
           },
         });
         break;
-      case 'create':
-        packet = {
+      case 'create': {
+        const packet = {
```

Comment:
It's bad practice to declare a variable inside a `case` statement, which is why this `packet` variable was declared above in the `let packet;` line